docs/content/guides/6.multistore/2.tooling-and-concepts/3.development/4.local-environment.md

```diff
@@ -7,6 +7,165 @@ navigation:
+
+During the development process, you will likely use the following scripts from the project root:
+- `dev`: Starts development mode for all stores
+- `build`: Creates production build for direct stores
+- `start`: Runs production build locally
+- `test:integration:pw`: Runs integration tests with Playwright
+
+## Starting Local Development
+
+The most common way to start development is using the yarn scripts from the root level:
+
+```bash
+# Start development mode for all stores
+yarn dev
+
+# Start development for a specific store
+yarn store dev --store-id=my-store
+```
+
+The `dev` command watches for changes in:
+- Base applications (`apps/storefront-unified-nextjs`, etc.)
+- Parent stores in the inheritance chain
+- The current store's files
+
+When a file changes, the system automatically:
+1. Detects which stores are affected by the change
+2. Moves the affected files to the `.out` directory, so the app's framework can hot-reload the changes
+
+::alert{type="info"}
+The applications will run on ports specified in the `alokai.config.json` file. For example:
+
+```json
+{
+  "stores": {
+    "default": {
+      "localPorts": {
+        "middleware": 4000,
+        "nextjs": 3000,
+        "nuxt": 3333
+      }
+    }
+  }
+}
+```
+
+Each store can have its own port configuration to avoid conflicts when running multiple stores simultaneously.
+::
+
+### Using Local Domains
+
+Alokai uses Caddy as a reverse proxy to handle local domains. The Caddyfile is automatically generated based on the `alokai.config.json` file and the `localDomain` property for each store.
+
+#### Prerequisites
+
+- [Caddy server installed](https://caddyserver.com/docs/install)
+- Local domains configured in your hosts file. For example:
+  - If you have a store with `localDomain` set to `default.local`, add `127.0.0.1 default.local` to your hosts file
+
+#### Starting with local domains
+
+Start all stores with their local domains:
+
+```bash
+yarn store dev --all --with-local-domains
+```
+
+:::tip HTTPS
+Caddy automatically generates self-signed SSL certificates for local domains. You 
+can read more about it in the [Caddy documentation](https://caddyserver.com/docs/automatic-https).
+:::
+
+## Building for Production
+
+To test the production build locally:
+
+1. Build all direct stores:
+```bash
+yarn build
+```
+
+2. Start the production server:
+```bash
+yarn start
+```
+
+Note that inheritance-only stores are not built into the `.out` directory, as they only serve as templates for their descendant stores.
+
+## Running Integration Tests
+
+Integration tests run against the composed stores, respecting the file inheritance chain:
+
+```bash
+yarn test:integration:pw
+```
+
+For more information on integration tests, refer to the [Integration Tests](/guides/multistore/tooling-and-concepts/development/local-environment) guide.
+
+## Common Issues and Solutions
+
+### Cache Problems
+
+If you experience unexpected behavior with file inheritance or turbo cache:
```

Comment:
Would be good to provide an example of what is unexpected, if possible :)

Reply:
Updated https://github.com/vuestorefront/vue-storefront/pull/7390/commits/96409d1f7f2a274c7a51b6daf1b940ba6e1d3933